session.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'time_calculator'
+
+class Session 
+  include TimeCalculator
+
+  def initialize(total_minutes, start_time)
+    @total_minutes = total_minutes
+    @start_time = start_time
+    @track = []
+  end
+
+  def start(talks)
+    talks.map do |talk| @total_minutes -= talk.duration
```

Comment:
If you're using do end, instead of lambda, then start the code in the next line.

Reply:
Done